e2e/_suites/metricbeat/metricbeat_test.go

```diff
@@ -154,18 +182,57 @@ func (mts *MetricbeatTestSuite) CleanUp() error {
 }
 
 func InitializeMetricbeatScenarios(ctx *godog.ScenarioContext) {
-	ctx.BeforeScenario(func(*messages.Pickle) {
-		log.Trace("Before scenario...")
+	ctx.BeforeScenario(func(p *messages.Pickle) {
+		log.Trace("Before Metricbeat scenario...")
+		if enableInstrumentation {
+			tx = apm.DefaultTracer.StartTransaction(p.GetName(), "test.scenario")
```

Comment:
wdyt about renaming using the scenario name _before_ interpolating the outline examples?

I.e. `<module> sends metrics to Elasticsearch without errors` Instead of `activemq-5.13.0 sends metrics to Elasticsearch without errors`

That way we will be able to aggregate information about same scenario steps. We can enrich the transaction with module name, and perhaps any of the outline example variables, as a label.